OrbitLinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -35,40 +37,60 @@ std::string ReadMaps(pid_t pid) {
   std::optional<std::string> maps_content_opt = ReadFile(maps_filename);
   if (maps_content_opt.has_value()) {
     return maps_content_opt.value();
-  } else {
-    return "";
   }
+  return "";
 }
 
-std::optional<std::string> ExecuteCommand(const std::string& cmd) {
-  std::unique_ptr<FILE, decltype(&pclose)> pipe{popen(cmd.c_str(), "r"), pclose};
-  if (!pipe) {
-    ERROR("Could not open pipe for \"%s\"", cmd.c_str());
-    return std::optional<std::string>{};
+static std::optional<pid_t> ProcEntryToPid(const std::filesystem::directory_entry& entry) {
```

Comment:
where is this pid_t coming from? Ad the client, we switched to int32_t for process/thread id. Does this also make sense to do so here?

Reply:
Okay, I see. Question would be if we might use int32_t anyways, to make it consistent across orbit. If yes, for sure in a different PR.

Reply:
I personally prefer using the type from the Linux headers in Linux-only code.

Reply:
Okay, fine for me.